app/build.gradle

```diff
@@ -11,7 +11,7 @@ android {
         // Make sure this is consistent with the redirect URI used in
         // res/raw/okta_app_auth_config.json
         manifestPlaceholders = [
-                "appAuthRedirectScheme": "com.okta.appauth.android.example"
+                "appAuthRedirectScheme": "com.lohika.android.test"
```

Comment:
i'm guessing this should still be `com.okta....` ?

Reply:
correct!